wordpress/plugins/kredeum-nfts/admin/media-list/actions.php

```diff
@@ -5,15 +5,16 @@
  * @package kredeum/nfts
  */
 
-namespace KredeumNFTs\Ipfs;
+namespace KredeumNFTs\Storage;
 
 /**
  * IPFS bulk archive action
  */
 add_filter(
 	'bulk_actions-upload',
 	function ( $bulk_actions ) {
-		$bulk_actions['archive'] = __( 'Archive to IPFS', 'kredeum-nfts' );
+		$bulk_actions['archive'] = __( 'Archive to ', 'kredeum-nfts' ) . strtoupper( STORAGE );
```

Comment:
replace witj plugin name ?  swarmpress or kredeum_nfts (define as constant ?)

Reply:
plugin name $domain arg in traduction function in wordpress must be a single string literal, not constant